### PR TITLE
added Acceptance tests

### PR DESCRIPTION
**Pull request by TheMSB**
Please proofread.

### CODE REVIEW

File: Report/Chapters/Testplan.tex [hunk 1]
```diff
@@ -6,8 +6,107 @@ \chapter{Test Plan}
...
+			\item Click on the item labelled Basic.
+			\item Observe that contents are present in the opened list.
+			\item Click on the Basic item again to close the list.
+			\item Press the button at the bottom of the menu labelled Value Block.
+			\item Enter "Hello" into the opened dialog window.
+			\item Confirm the presence of a block labelled "Hello" in the workspace.
+		\end{enumerate}
+		
+	\subsection{Test 2}
+		This test is a follow up to Test 1 and will evaluate the functioning of the context menu
```
Comment (wandernauta) [inline Report/Chapters/Testplan.tex]: Does that mean the steps from Test 1 have to be executed first?

File: Report/Chapters/Testplan.tex [hunk 2]
```diff
@@ -6,8 +6,107 @@ \chapter{Test Plan}
...
+	\subsection{Test 1}
+		This test evaluates the functioning of the FunctionMenu and the spawning of basic blocks.
+		
+		\begin{enumerate}
+			\item Right click on the workspace to open the FunctionMenu.
+			\item Click on the item labelled Basic.
+			\item Observe that contents are present in the opened list.
+			\item Click on the Basic item again to close the list.
+			\item Press the button at the bottom of the menu labelled Value Block.
+			\item Enter "Hello" into the opened dialog window.
```
Comment (wandernauta) [inline Report/Chapters/Testplan.tex]: Might need to clarify that it should be entered with double quotes.

File: Report/Chapters/Testplan.tex [hunk 3]
```diff
@@ -6,8 +6,107 @@ \chapter{Test Plan}
...
+			\item Click on the item labelled "Numeric Types".
+			\item Observe that contents are present in the opened list.
+			\item Select the entry in the list labelled "(+)".
+			\item Confirm that a FunctionBlock titled "(+)" has been made in the workspace.
+			\item Connect the output of the Value Block with the FunctionBlock.
+			\item Confirm that the argument in the FunctionBlock and the Connection line are highlighted red.
+		\end{enumerate}
+	
+	\subsection{Test 5}
+		This test will evaluate the functionality of higher order functions, bow ties and GraphBlocks as defined in Viskell.
```
Comment (wandernauta) [inline Report/Chapters/Testplan.tex]: Official terminology has changed to 'knots', I believe.

File: Report/Chapters/Testplan.tex [hunk 4]
```diff
@@ -6,8 +6,107 @@ \chapter{Test Plan}
...
+		
+		\begin{enumerate}
+			\item Right click on the workspace to open the FunctionMenu.
+			\item Press the button at the bottom of the menu labelled Graph Block.
+			\item Select the entry in the FunctionMenu labelled ?Numeric Types?.
```
Comment (wandernauta) [inline Report/Chapters/Testplan.tex]: This shows on GitHub as `?Numeric Types?', is that correct?

File: Report/Chapters/Testplan.tex [hunk 5]
```diff
@@ -6,8 +6,107 @@ \chapter{Test Plan}
...
+		\begin{enumerate}
+			\item Right click on the workspace to open the FunctionMenu.
+			\item Select the entry in the FunctionMenu labelled ?Numeric Types?.
+			\item Select the (*) function to create a new FunctionBlock.
+			\item	Scroll down in the newly opened list until you find the Sin function.
+			\item Click on the item to create a new Sin FunctionBlock.
+			\item Press the button at the bottom of the menu labelled Graph Block.
+			\item Press the button at the bottom of the menu labelled Slider Block.
+			\item Press the button at the bottom of the menu labelled Definition Block.
+			\item In the opened dialoge window input: ?trans :: Float -> Float? and press enter.
```
Comment (wandernauta) [inline Report/Chapters/Testplan.tex]: `dialog`, no e.

**Comment by wandernauta** [comment]
Looks like a good addition. Thanks a lot! Will merge with comments fixed.

File: Report/Chapters/Testplan.tex (hunk 1 above)
Comment (TheMSB) [inline Report/Chapters/Testplan.tex]: Yes

File: Report/Chapters/Testplan.tex (hunk 2 above)
Comment (TheMSB) [inline Report/Chapters/Testplan.tex]: Suggestions as to how?

File: Report/Chapters/Testplan.tex (hunk 5 above)
Comment (TheMSB) [inline Report/Chapters/Testplan.tex]: fixed

File: Report/Chapters/Testplan.tex (hunk 3 above)
Comment (TheMSB) [inline Report/Chapters/Testplan.tex]: fixed

File: Report/Chapters/Testplan.tex (hunk 2 above)
Comment (wandernauta) [inline Report/Chapters/Testplan.tex]: Something like this?

    Enter "Hello" into the opened dialog window (including quotation marks).

File: Report/Chapters/Testplan.tex (hunk 4 above)
Comment (wandernauta) [inline Report/Chapters/Testplan.tex]: Probably an encoding issue, will fix this for you when I merge this.

**Comment by wandernauta** [comment]
Merged. Thanks again! :sparkles:

**Pull request merged**